src/seedu/addressbook/data/person/UniquePersonList.java

```diff
@@ -129,6 +128,14 @@ public void clear() {
         internalList.clear();
     }
 
+    /**
+     * Sort lists alphebetically
+     */
+    public void sort()
```

Comment:
While it may seems trivial to write a test for this, however if you are to implement sorting with order (ascending/descending), it will be useful to have a test written to guarantee the default order is as what is intended.

Reply:
Noted! Thanks!